R/showLinePlots.R

```diff
@@ -28,55 +29,42 @@
 #' showLinePlots(data, "Policy Cost|GDP Loss")
 #' }
 #' @export
-#' @importFrom rlang .data .env
 #' @importFrom dplyr bind_rows
 #' @importFrom gridExtra arrangeGrob
+#' @importFrom quitte as.quitte getVars
+#' @importFrom rlang .data .env
 
 showLinePlots <- function(
     data,
-    vars = NULL,
-    histVars = NULL,
+    vars = getVars(as.quitte(data)),
+    histVars = vars,
```

Comment:
I have never tested this. Would R know to set the default of histVars to the default of vars, if neither is given?

Reply:
```r
foo <- function(d, vars = d[['variable']], histVars = rev(vars))
{
    list(vars, histVars)
}

d <- data.frame(variable = LETTERS[1:3])

foo(d)
# [[1]]
# [1] "A" "B" "C"
# 
# [[2]]
# [1] "C" "B" "A"
```